web/src/components/network/AddressesDataList.jsx

```diff
@@ -118,24 +114,16 @@ export default function AddressesDataList({
   const newAddressButtonText = addresses.length ? "Add another address" : "Add an address";
 
   return (
-    <Stack className="data-list-form" hasGutter>
-      <StackItem>
-        <Split hasGutter>
-          <SplitItem isFilled>
-            <FormLabel isRequired={!allowEmpty}>Addresses</FormLabel>
-          </SplitItem>
-          <SplitItem>
-            <Button isSmall variant="secondary" className="btn-sm" onClick={() => addAddress()}>
-              {newAddressButtonText}
-            </Button>
-          </SplitItem>
-        </Split>
-      </StackItem>
-      <StackItem>
-        <DataList isCompact gridBreakpoint="none" title="Addresses data list">
-          {addresses.map(address => renderAddress(address))}
-        </DataList>
-      </StackItem>
-    </Stack>
+    <>
+      <div className="split justify-between">
+        <FormLabel isRequired={!allowEmpty}>Addresses</FormLabel>
+        <Button isSmall variant="secondary" className="btn-sm" onClick={() => addAddress()}>
```

Comment:
If you prefer, I think you can write: `onClick={addAddress}`. However, it is out of the scope of this PR.

Reply:
Yep, you're right. To be honest, I didn't realize about it because I was focus in the UI topic. Let's improve it in another PR.

Anyway, it's interesting to see that there is a _className="btn-sm"_ leftover there. And it was added by me in https://github.com/yast/d-installer/pull/391, probably because it's a component borrowed from cockpit-wicked where we [introduced that CSS class](https://github.com/openSUSE/cockpit-wicked/blob/f26f4de86b258ccabc77857a179a93c37746695f/src/app.scss#L46-L49) :man_shrugging: :man_shrugging: :man_shrugging:

Reply:
Finally, I have changed those onClick function to their shorter versions since I had to edit the file for removing the leftover CSS.